crates/networking/rpc/Cargo.toml

```diff
@@ -26,6 +26,7 @@ rand.workspace = true
 tokio-util.workspace = true
 reqwest.workspace = true
 k256 = { version = "0.13.3", features = ["ecdh"] }
+libsecp256k1 = "0.7.1"
```

Comment:
Can we put this on root's `Cargo.toml` and use `.workspace = true` here?

Reply:
Yep, doing it asap.

Reply:
Done [57a19d8](https://github.com/lambdaclass/ethrex/pull/2274/commits/57a19d8a3d044211bda4cc076db51762aefe900d)